### ISSUE TITLE
Regarding external npm packages

### ISSUE THREAD

**Issue by Joelgullander** [gregberge/loadable-components#566]
## 💬 Questions and Help

Hey guys, really great library. I just got a few architectural questions.. We are using a react client as a frontend and have structured all our components / functions into a lerna mono repo. As a best practice, would we export the components with loadable from the components npm library or export them as usual and handle the loadable-components where the components are consumed ( in the react frontend )?

**Comment by theKashey** [comment]
That's really not quite correct question.
If you can consume one "lazy" component, which will consume another "lazy" component - then you will eventually consume "lazy" component as a part of another one. In this case it does not really matters where you put loadable - insite another package, or outside - you will have both cases  in the final code.

However, the decision about "how something should be loaded" is usually up to consumer. The placement of Suspense boundary and ErrorBoundary is up to the consumer as well.
This is also bound to the ability to `precache` or `preload` - package itself usually don't not know how and why it will be consumed, however it depends on the package.

In other words - there is no answer to your questions. Just no answer.
